### PR TITLE
Some improvements to horizontal layout

### PR DESCRIPTION
**Pull request by konstantin-mohin**
@igala still don't get why we have additional 50.0 line 765

### CODE REVIEW

File: galaxy_mobile/lib/screens/streaming/streaming.dart [hunk 1]
```diff
@@ -782,26 +786,28 @@ class _StreamingUnifiedState extends State<StreamingUnified> {
...
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           Container(
-            height: getHeight() - 50,
+            height: getHeight(),
             width: getWidth(),
             decoration: BoxDecoration(
                 border: Border.all(
                     color:
-                        widget.isOnAir ? Colors.redAccent : Colors.transparent,
```
Comment (igala) [inline galaxy_mobile/lib/screens/streaming/streaming.dart]: the border should be transparent and not green

**Review by igala — CHANGES_REQUESTED** [review]
looks good , just fix the border color

File: galaxy_mobile/lib/screens/streaming/streaming.dart (hunk 1 above)
Comment (konstantin-mohin) [reply galaxy_mobile/lib/screens/streaming/streaming.dart]: @igala yeah, right,  was debug with this color

**Comment by konstantin-mohin** [comment]
@igala pushed

**Pull request merged**